radio/src/gui/9X/menu_model_telemetry.cpp

```diff
@@ -606,7 +606,7 @@ void menuModelTelemetry(uint8_t event)
     switch (k) {
 #if defined(CPUARM)
       case ITEM_TELEMETRY_PROTOCOL_TYPE:
-        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
+        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (inv)\0 FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
```

Comment:
You are right, I had not considered that. Will change. Cleaning up as part of the move to 2.2 sounds like a good plan too.

When I tried to build `next` with my settings it failed, that's why I wrote my patch against `main`. Is there documentation on how features are merged / grafted between `main` and `next`, or what is a good place to ask some questions about that, and the release process?

As an alternative to the implementation of 'FrSky D (inverted)' as a separate 'protocol', there would be the option to add a checkbox 'invert internal serial port' and make this independent of the protocol selection. Thoughts?